packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1756,28 +1703,30 @@ export class ContainerRuntime
 
 		this.handleContext = new ContainerFluidHandleContext("", this);
 
-		if (this.summaryConfiguration.state === "enabled") {
-			this.validateSummaryHeuristicConfiguration(this.summaryConfiguration);
+		if (summaryConfiguration.state === "enabled") {
+			this.validateSummaryHeuristicConfiguration(summaryConfiguration);
 		}
 
-		this.summariesDisabled = this.isSummariesDisabled();
-		this.maxOpsSinceLastSummary = this.getMaxOpsSinceLastSummary();
-		this.initialSummarizerDelayMs = this.getInitialSummarizerDelayMs();
+		this.summariesDisabled = isSummariesDisabled(summaryConfiguration);
+		const {
+			maxOpsSinceLastSummary = 0,
+			// back-compat: initialSummarizerDelayMs was moved from ISummaryRuntimeOptions
+			//   to ISummaryConfiguration in 0.60.
+			initialSummarizerDelayMs = runtimeOptions.summaryOptions.initialSummarizerDelayMs ?? 0,
+		} = isSummariesDisabled(summaryConfiguration) ? {} : summaryConfiguration;
```

Comment:
This is terse but kind of confusing to follow as the assignment precedence bounces between lines.  Looks like this has resulted in summaryConfiguration overriding summaryOptions for initialSummarizerDelayMs, a behavior change since currently summaryOptions takes precedence?

Reply:
summaryConfiguration is a really hard type to work with, so i'm not surprised the deprecated field is still around and used. i fixed precedence issue. i would like to fix the type, but is it exposed, so hard to make changes.

Reply:
None of either summaryConfiguration nor summaryOptions is directly exposed publicly AFAICT, and since the class is now hidden maybe there's some opportunity to precompute these in loadContainerRuntime such that the ctor can just take a simple initialSummarizerDelayMs directly.  Might be nice to (1) externalize that complication from the class, (2) pass the tough objects through as few calls as possible, and (3) probably makes the class simpler to test directly?  Not needed for this PR but something to think about.